mpmaps/__init__.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+from platformdirs import user_data_dir
+import urllib.request
+import os
+
+this_file = os.path.basename(__file__)
+FILE_DIR = Path(this_file).resolve().parent
+download_path = os.path.join(FILE_DIR, "mpgrids")
```

Comment:
FILE_DIR is a `Path` object in this context

so you can just do 

```
download_path = FILE_DIR / "mpgrids"
```

if you care, not sure it matters